src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -27,11 +27,10 @@ public JsonSchemaTransformer(JsonSchema schema)
             return (errors, transformedToken);
         }
 
-        public (List<Error> errors, Dictionary<string, (bool, Dictionary<string, Lazy<JToken>>)> properties) TransformXref(Document file, Context context, JToken token)
+        public (List<Error> errors, Dictionary<string, List<(bool isRoot, string jsonPath, Dictionary<string, Lazy<JToken>> propertiesByUid)>> properties) TransformXref(Document file, Context context, JToken token)
```

Comment:
List<string, (bool, Dictionary<string, Lazy<JToken>>)>

Reply:
Actually, it would be the same, with `List<...>`, we have to group by `uid` from the caller.